src/app/shared/components/fy-approver/add-approvers-popover/approver-dialog/approver-dialog.component.ts

```
@@ -166,7 +166,7 @@ export class ApproverDialogComponent implements AfterViewInit, OnInit {
     } else {
       return this.getDefaultUsersList().pipe(
         switchMap((employees) => {
-          employees = employees.filter((employee) => this.approverEmailsList.indexOf(employee.us_email) === -1);
+          employees = employees.filter((employee) => this.approverEmailsList.indexOf(employee.email) === -1);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)